src/main/java/de/zalando/aruha/nakadi/service/subscription/model/Session.java

```diff
@@ -0,0 +1,42 @@
+package de.zalando.aruha.nakadi.service.subscription.model;
+
+import java.util.UUID;
+
+public class Session {
+    public final String id;
```

Comment:
I use public fields if:
 - the field is final
 - the actual value that field is holding is not changed in time.

Reasons: 
 - I don't see any reasons to create getters that will just return field value.